pyproject.toml

```diff
@@ -33,7 +33,7 @@ known_local_folder = ["noisy_search", "remote", "sighandler"]
 [tool.pytest.ini_options]
 testpaths = "test"
 addopts = "--cov-context=test --cov-report html --import-mode=prepend"
-markers = ["mask_timestamp", "use_vkgdr", "cmdline_args"]
+markers = ["mask_timestamp", "use_vkgdr", "cmdline_args", "spectra_per_heap"]
```

Comment:
Admittedly not 100% sure on how these markers fit in with running pytest (I know I could go off and look it up).

Does this allow you to override the args from your own `pytest.ini` file?

Reply:
Ah, I see you can `@pytest.mark` any of these args at the top of a unit test. Nice one.

Reply:
Yes. Putting them here just tells pytest "yes this really is a mark that has some meaning." Otherwise it'll warn you that you might be missing a plugin that would interpret it.